### PR TITLE
ToS acceptance tracking: Update the file upload endpoint

### PR DESCRIPTION
**Pull request by niranjan-uma-shankar**
#### Changes proposed in this Pull Request

* Update the file upload endpoint used for the screenshots zip file. The original PR that introduced the screenshot feature is in #57741. 
* This change is as per the advise given in pMz3w-eJp#comment-96860 to reuse the existing file upload endpoint.

#### Testing instructions

* Check out this branch to your local instance
* In test/e2e folder of your local Calypso folder, run yarn install
* Apply D78318-code and follow the setup instructions given in the "Test Plan" of that patch.
* In the `test/e2e` folder, run `BABEL_ENV=test yarn jest specs/martech/tos-screenshots__checkout.ts` and also the other specs in the martech folder.
* Verify that fullpage screenshots of signup, login, and checkout are stored on the blog, and the ToS link is visible in all these pages.

Related to #

### CODE REVIEW

File: test/e2e/specs/martech/tos-screenshots__checkout.ts
Comment (worldomonation) [inline test/e2e/specs/martech/tos-screenshots__checkout.ts]: Style-wise, the preference is to use individual calls over `then` actions where possible. Is that possible here and elsewhere that has similar construct?

**Review by worldomonation — APPROVED** [review]
Generally the changes look good to me. 

Approving for merge once you decide on a course of action for the comment I left.

File: test/e2e/specs/martech/tos-screenshots__checkout.ts
Comment (niranjan-uma-shankar) [reply test/e2e/specs/martech/tos-screenshots__checkout.ts]: @worldomonation I've refactored the code to remove the `then` actions. Is this what you had in mind?

File: packages/calypso-e2e/src/data-helper.ts
Comment (worldomonation) [inline packages/calypso-e2e/src/data-helper.ts:176]: I think this should be included in the encrypted secrets file if possible, but there's an active PR (https://github.com/Automattic/wp-calypso/pull/63183) to refactor the entire secrets file. 

At this moment, I think we can keep the raw siteID in code, but as soon as the other PR lands I will look into moving this into the secrets.

Comment (niranjan-uma-shankar) [reply packages/calypso-e2e/src/data-helper.ts:176]: Thanks @worldomonation . ~If the PR otherwise looks good, could you please approve it?~  Edit: Noticed that you'd already approved the PR.

**Pull request merged**
